api/server/router/image/image_routes.go

```diff
@@ -241,6 +241,7 @@ func (s *imageRouter) postImagesLoad(ctx context.Context, w http.ResponseWriter,
 	output := ioutils.NewWriteFlusher(w)
 	defer output.Close()
 	if err := s.backend.LoadImage(r.Body, output, quiet); err != nil {
+		output.WriteHeader(http.StatusUnprocessableEntity)
```

Comment:
Yes, was doubting to make it more generic (just a `400`), to prevent having to do string-matching here for other errors.